modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/confirm.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export function confirm({message, onConfirm}) {
+	Liferay.Util.openModal({
+		bodyHTML: message,
+		buttons: [
+			{
+				displayType: 'secondary',
+				label: Liferay.Language.get('cancel'),
+				type: 'cancel',
+			},
+			{
+				label: Liferay.Language.get('ok'),
+				onClick: ({processClose}) => {
+					processClose();
+
+					onConfirm(true);
```

Comment:
Do we need the return value? I would assume confirm has only one handler, for positive outcome. In your code example, this would be:

```js
function handleConfirm() {
    // no if
    // run some logic here 
}

Liferay.Util.confirm('are you sure, Marko?', handleConfirm)
```

We use vanilla `confirm` in many places in DXP. Is there a single case where we have `else`? Something like
```js
if (confirm(...)) {
    // run some logic here 
} else {
   // run some logic on cancel
}
```

Reply:
> Do we need the return value? 

Having a callback that will look like just a hook sounds weird to me but I think it's more related to my previous experiences/background. If you want we can drop and call this onConfirm only when `true`. 

> Is there a single case where we have else?

I searched for these use case in js files and yes, we are not using else. Maybe is related to early return pattern that we are adopting in recent files? My fear is to miss a use case that we are not using [early return](https://forum.freecodecamp.org/t/the-return-early-pattern-explained-with-javascript-examples/19364) pattern for conditionals.

I would like to maintain the received value from the callback to detect if it's `true` or `false` :/

Reply:
Edit: I found one use case containing `else`: https://github.com/liferay/liferay-portal/blob/444c055c5fe89f2eded9d56934c019612d575a56/modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/js/WikiPortlet.es.js#L191 :p

Reply:
ok, let's keep the return value :+1: